Sources/EmbraceOTelInternal/Logs/EmbraceLogRecordBuilder.swift

```diff
@@ -55,6 +56,10 @@ class EmbraceLogRecordBuilder: EventBuilder {
         return self
     }
 
+    func setData(_ attributes: [String: OpenTelemetryApi.AttributeValue]) -> Self {
+        return setAttributes(attributes)
+    }
+
```

Comment:
Seems like implementing this right now is kinda useless. Probably this is needed because when I've created the `EmbraceLogRecordBuilder` I've made the class conformant to `EventBuilder` instead of implementing directly `LogRecordBuilder` (note: `EventBuilder` inherits from `LogRecordBuilder`.
I think we could be able to get rid of it

Reply:
What would we return in `EmbraceLogger` `eventBuilder(name: String)` then?

Reply:
Mmm I see why we did it then 😂 . You're right. We should leave it as is then.
Changing the `EmbraceLogger.eventBuilder(name: String)` to use other thing (like the default `LogRecordBuilderSdk`) would require other changes and seems that is not worth it.